### PR TITLE
Feature/form validation

### PR DESCRIPTION
**Pull request by Fadahunsiseyi**
### Form Validation.

- I added the form validation.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -148,24 +148,26 @@ <h3 class="title">Skills</h3>
...
+               <input type="email"  id="userEmail" class="username" placeholder="Enter your Email">
             </li>
-            <li>
-              <textarea name="message" id="textarea-message" cols="30" rows="10" maxlength="500" placeholder="Write your message"></textarea>
+            <li class="form-field">
+              <small></small>
+              <textarea class="username" id="textareaMessage" cols="30" rows="10" maxlength="500" placeholder="Write your message"></textarea>
             </li>
             <li>
               <button type="submit" class="submit-btn">Get in Touch</button>
```
Comment (tufoinnkuo10) [inline index.html:173]: - [ ]great job with the error message. kindly make the error message show up beside the submit button to easily be noticed. 
Screenshot (167)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Fadahunsiseyi  and @altontonn 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ email check is case sensitive
✅ error message shows up

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
- [ optional] its a pair program project, kindly avoid words like `I ` in your PR summary
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Fadahunsiseyi** [comment]
Hi Code Reviewer,

I am not clear with the review. You stated the email should be in caps. I
didn't get the correction for the error beside the button

File: index.html [hunk 2]
```diff
@@ -158,18 +158,18 @@ <h2 class="contact-form-heading">Contact Me</h2>
...
+               <small></small>
             </li>
             <li class="form-field">
-              <small></small>
               <textarea class="username" id="textareaMessage" cols="30" rows="10" maxlength="500" placeholder="Write your message"></textarea>
+              <small></small>
             </li>
-            <li>
+            <li class="form-field">
               <button type="submit" class="submit-btn">Get in Touch</button>
```
Comment (topeogunleye) [inline index.html:173]: - [ ] As suggested by the previous reviewer. Please kindly move the error message close to the submit button:

image

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Fadahunsiseyi.
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @Fadahunsiseyi, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**